gulpfile.js

```diff
@@ -192,33 +218,26 @@ gulp.task('styles-grid', function() {
     .pipe($.size({title: 'styles-grid'}));
 });
 
+// Build with Google's Closure Compiler.
+// To run this step, run "bower install" within MDL to fetch the compiler jar,
+// note that Java 1.7+ is required. This is an optional step.
+gulp.task('closure', function() {
+  return gulp.src(SOURCES)
+    .pipe(closureCompiler({
+      compilerPath: 'bower_components/closure-compiler/compiler.jar',
+      fileName: './dist/mdl.closure.min.js',
+      compilerFlags: {
+        // jscs:disable requireCamelCaseOrUpperCaseIdentifiers
```

Comment:
We now use the custom `closureCamelCase` rule, you'll need to replace that one in here.